workbench-core/example/app/src/configs/staticRouteConfig.ts

```diff
@@ -37,11 +38,42 @@ export const routesMap: RoutesMap = {
       }
     ]
   },
-  '/datasets/share': {
+  [`/datasets/${dataSetPrefix.toLowerCase()}-${uuidRegExpAsString}`]: {
+    GET: [
+      {
+        action: 'READ',
+        subject: 'Dataset'
+      }
+    ],
+    DELETE: [
+      {
+        action: 'DELETE',
+        subject: 'Dataset'
+      }
+    ]
+  },
+  [`/datasets/${dataSetPrefix.toLowerCase()}-${uuidRegExpAsString}/share`]: {
     POST: [
       {
         action: 'CREATE',
-        subject: 'Dataset'
+        subject: 'Endpoint'
+      }
+    ]
+  },
+  [`/datasets/${dataSetPrefix.toLowerCase()}-${uuidRegExpAsString}}/share/${endPointPrefix.toLowerCase()}-${uuidRegExpAsString}`]:
+    {
+      DELETE: [
+        {
+          action: 'DELETE',
+          subject: 'Endpoint'
+        }
+      ]
+    },
+  '/datasets/storage': {
+    GET: [
+      {
+        action: 'READ',
+        subject: 'Storage'
```

Comment:
Is this from your listStorageLocations integ tests work?

Reply:
yes